app/routes/application_status.py

```diff
@@ -4,19 +4,19 @@
 
 from ..libs import VivaApplicationStatus
 from ..libs import decode_hash_id
-from ..errors import ValidationError
+from ..errors import CustomValidationError
 
 
 class ApplicationStatus(Resource):
 
     def get(self, hash_id):
-        hash_id_tuple = decode_hash_id(hash_id=hash_id)
+        decoded_hash_id = decode_hash_id(hash_id=hash_id)
 
-        if len(hash_id_tuple) == 0:
-            raise ValidationError(
-                message=f'The hashid {hash_id} in the url is not a valid hashid')
+        if len(decoded_hash_id) == 0:
+            raise CustomValidationError(
+                message=f'The server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing).')
```

Comment:
Man... Hope this never happens. The web version 2000 comes out before I have read the error message.

Reply:
This is the default Bad Request Error description, so not my own words. But if you have any better idea, i'm open for suggestions.